cmake/Findnl-genl-3.cmake

```diff
@@ -0,0 +1,32 @@
+# SPDX-License-Identifier: BSD-2-Clause-Patent
+#
+# Copyright (c) 2019 Intel Corporation
+#
+# This code is subject to the terms of the BSD+Patent license.
+# See LICENSE file for more details.
+
+find_library(NL3_GENL_LIBRARY "libnl-genl-3.so")
+find_path(NL3_INCLUDE_DIRS libnl3/netlink/genl/genl.h)
+message(NL3_INCLUDE_DIRS=${NL3_INCLUDE_DIRS})
+
+include(FindPackageHandleStandardArgs)
+FIND_PACKAGE_HANDLE_STANDARD_ARGS(nl-genl-3 DEFAULT_MSG
+    NL3_GENL_LIBRARY
+    NL3_INCLUDE_DIRS
+)
+
+if (nl-genl-3_FOUND)
+    add_library(nl-genl-3 UNKNOWN IMPORTED)
```

Comment:
I believe nl-genl-3 depends on nl-3. I'm not entirely sure if we need to encode that though.